src/components/Button/Button.tsx

```diff
@@ -1,16 +1,12 @@
 import clsx from 'clsx';
 import React, { MouseEventHandler, ReactNode } from 'react';
 import styles from './Button.module.css';
-import { Icon } from '../Icon/Icon';
+
 export interface Props {
   /**
    * Button reference
    */
   buttonRef?: any;
```

Comment:
I'm curious why this is called `buttonRef` instead of `ref`. Is there a chance devs will think it's referring to the icon?

Reply:
It's because you can't name a prop `ref`. React actually has special handling for `ref` and `key`, which look like props but are handled differently.

The only way to get something like `<Button ref={someRef} />` is to use `forwardRef`. But an alternative to that is to just name a prop something other than `ref`. Like `buttonRef`.

Reply:
Ah right, thanks!